pdr_backend/pdr_dashboard/pages/predictoors.py

```diff
@@ -87,10 +87,8 @@ def get_search_bar_row(self):
         )
 
     def get_main_container(self):
-        predictoor_stats = self.app.db_getter.predictoor_payouts_stats()
-
         predictoor_cols, predictoor_data, raw_predictoor_data = (
-            self.get_data_for_predictoors_table(predictoor_stats)
+            self.get_data_for_predictoors_table(self.app.predictoors_data)
```

Comment:
you can use the same self.app.predictoors data in self.get_data_for_predictoors_table, since it's the same instance. No need to pass it around as a parameter